cherryontech/birthdaybasher
# Add tailwind

**Work done**

This PR adds the Tailwind packages, as per the [Girdsome Tailwind docs](https://next.gridsome.org/plugins/gridsome-plugin-tailwindcss), and the [Tailwind compat docs](https://tailwindcss.com/docs/installation#post-css-7-compatibility-build).

**Testing instructions**
Run gridsome locally (see package.json for scripts you can use).
Add a tailwind class to any element, and check it in the browser. The proper Tailwind styles should be applied.
